src/app/components/Slideout/Users/SlideoutUsername.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <li
+    @mouseover="hover = true"
+    @mouseleave="hover = false"
+  >
+    <Username :user="user"
+      @focus="hover = true" />
+      <div style="overflow:hidden">
+      <button
+        class="chat-message-options border-0"
+        ref="chatOptions"
+        @click.prevent="openContextMenu"
+        v-show="hover && user.username"
+        @blur="hover = false"
+        @keypress.enter.prevent="openContextMenuWithKey"
+        tabindex=0
+      >
+        &vellip; <!-- ⋮ -->
+      </button>
+      <ActionMenu
+        ref="contextMenu"
+        :message="messageFrom(user)"
+      />
+    </div>
+  </li>
+</template>
+<script lang="ts">
+  /* This is essentially a duplicate of UserMessage, just with the message removed */
```

Comment:
Seems like this calls for a generic component for both the card in the slideout and when hovering over a user in the chat?